wire-ios/Wire-iOS/Sources/UserInterface/DeviceView/DeviceInfoViewModel.swift

```diff
@@ -149,15 +150,10 @@ final class DeviceInfoViewModel: ObservableObject {
         }
     }
 
+    @MainActor
     func removeDevice() async {
```

Comment:
Isn't it sufficient to mark the observed property `shouldDissmissView` with @MainActor instead of the whole func?

Reply:
No Need to call the underlying call to remove device as well to be run on main thread